SummerProject/article_db/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from ..user_auth.models import User
+
+from enum import Enum
+
+class SocialNetworkChoice(Enum):
+    FB = "Facebook"
+    TR = "Twitter"
+    GGL = "Google"
+
+class Category(models.Model):
+    name = models.CharField(max_length=155, blank=False)
+    parent_category = models.ForeignKey('self', on_delete=models.CASCADE)
+
+class Team(models.Model):
+    name = models.CharField(max_length=150)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+
+class Article(models.Model):
+    headline = models.CharField(max_length=150, blank=False)
+    photo = models.ImageField(max_length=150)
+    video = models.URLField(max_length=150)
+    author = models.CharField(max_length=55, blank=False)
+    source = models.CharField(max_length=3, choices=[(tag, tag.value) for tag in SocialNetworkChoice])
+    content = models.TextField(blank=False)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+
+class Comment(models.Model):
+    content = models.CharField(max_length=1000)
+    date = models.DateTimeField()
```

Comment:
You could make use of Django's `auto_now_add` here -https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.DateField.auto_now_add

Reply:
Done